packages/node_modules/@node-red/editor-client/src/js/ui/deploy.js

```diff
@@ -421,7 +421,7 @@ RED.deploy = (function() {
 
             const unusedConfigNodes = [];
             RED.nodes.eachConfig(function (node) {
-                if ((node._def.hasUsers !== false) && (node.users.length === 0)) {
+                if ((node._def.hasUsers !== false) && (node.users.length === 0) && (node.type !== "global-config")) {
```

Comment:
```suggestion
                if ((node._def.hasUsers !== false) && (node.users.length === 0)) {
```

We do not need to add a special case here for `global-config` if we add `hasUsers: false` to its definition in `91-global-config.html`